workflow/hpxml_inputs.json

```diff
@@ -1003,7 +1003,9 @@
   },
   "sample_files/base-battery-scheduled-power-outage.xml": {
     "parent_hpxml": "sample_files/base-battery-scheduled.xml",
-    "schedules_power_outage_periods": "Jul 1 5 - Jul 31 14"
+    "schedules_unavailable_period_types": "Power Outage",
+    "schedules_unavailable_period_dates": "Jul 1 5 - Jul 31 14",
+    "schedules_unavailable_period_window_natvent_availabilities": "regular schedule"    
```

Comment:
Why do you have to provide the `window_natvent_availabilities` argument now? What was it doing before?

Reply:
I'm now requiring that the 3 new arguments are either (1) all not initialized or (2) all initialized. And, if `schedules_unavailable_period_window_natvent_availabilities` is initialized, it must contain only valid ventilation types.